### PR TITLE
active dot style customizable by prop

### PR DESCRIPTION
**Pull request by cure-tmk**
Made active dot style customizable by handing a `activeDotStyle`(optional) prop.

```jsx
<Carousel
    autoSlideInterval={3000}
    useDots={true}
    useControl={true}
    activeDotStyle={{
        'background': '#F00'
    }}
>
```
this will render the following.

518c732a5f7d31a609c1446aa0a11fa8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,4 +84,3 @@ npm start
 ```sh
 npm test
 ```
-
```
Comment (RyotaSugawara) [inline README.md]: do not commit this change, plz

File: docs/index.tsx [hunk 2]
```diff
@@ -73,4 +73,3 @@ ReactDOM.render(
   <App />,
   document.getElementById('app')
 );
-
```
Comment (RyotaSugawara) [inline docs/index.tsx]: same as
https://github.com/RyotaSugawara/react-isomorphic-carousel/pull/6/files#r213161980

Comment (cure-tmk) [reply docs/index.tsx]: fixed.

File: src/index.tsx [hunk 3]
```diff
@@ -150,6 +161,16 @@ export default class Carousel extends React.Component<CarouselProps, CarouselSta
...
+    const BUTTON_STYLE = {
+      ...DEFAULT_BUTTONS_STYLE,
+      background: '#ccc',
+      ...this.props.dotStyle
+    }
+    const ACTIVE_BUTTON_STYLE = {
+      ...DEFAULT_BUTTONS_STYLE,
+      background: '#000',
+      ...this.props.activeDotStyle
+    }
```
Comment (RyotaSugawara) [inline src/index.tsx]: I want to declare this style only once.
or create a class method to get this style.

Comment (cure-tmk) [reply src/index.tsx]: fixed

**Review by RyotaSugawara — APPROVED** [review]
LGTM

**Pull request merged**